tools/changelog/changelog.go

```diff
@@ -0,0 +1,144 @@
+/*
+Copyright 2024 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"regexp"
+	"strings"
+	"text/template"
+
+	"github.com/gravitational/shared-workflows/libs/gh"
+	"github.com/gravitational/trace"
+)
+
+const (
+	// timeNow is a convenience variable to signal that search should include PRs up to current time
+	timeNow = ""
+)
+
+// changelogInfo is used for the changelog template format.
+type changelogInfo struct {
+	// Summary is the changelog summary extracted from a PR
+	Summary string
+	Number  int
+	URL     string
+}
+
+const (
+	ossCLTemplate = `
+{{- range . -}}
+* {{.Summary}} [#{{.Number}}]({{.URL}})
+{{ end -}}
+`
+	entCLTemplate = `
+{{- range . -}}
+* {{.Summary}}
```

Comment:
Why does the enterprise template not include PR number/URL? External users can't access those PRs but I think we should still include it, it's useful for us.

Reply:
I removed it as it was indicated in the issue to remove it since it wasn't useful to external users. I had noticed that these links are also not present in the some of the newer Enterprise entries in the changelog.

Reply:
I think we should keep including them.